src/test/java/uk/gov/hmcts/reform/civil/service/NotificationServiceTest.java

```diff
@@ -0,0 +1,66 @@
+package uk.gov.hmcts.reform.civil.service;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.jackson.JacksonAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import uk.gov.hmcts.reform.civil.helpers.CaseDetailsConverter;
+import uk.gov.service.notify.NotificationClient;
+import uk.gov.service.notify.NotificationClientException;
+
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@SpringBootTest(classes = {
+    NotificationService.class,
+    JacksonAutoConfiguration.class,
+    CaseDetailsConverter.class
+})
```

Comment:
This notation is more appropriate for integration tests.

Use the standard JUnit5/mockito approach.

I'll show you how separately, as it involves some refactoring of the below as well.